cli/console.ts

```diff
@@ -223,6 +223,17 @@ export function printExecutionGraph(executionGraph: dataform.ExecutionGraph, asJ
   }
 }
 
+export function formatExecutionSuffix(jobIds: string[], bytesBilled: string[]): string {
```

Comment:
Let's move it to utils as well, I think it would be cleaner than export it from here for the testing purpose only.